src/blockchain_db/blockchain_db.cpp

```diff
@@ -26,6 +26,8 @@
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#ifndef APPLE_DEBUG_BLOCKCHAIN_DB
```

Comment:
Similarly, this `#ifndef` wrap prevents the compilation of the whole of `blockchain_db.cpp`, save for 

```
namespace blockchain_db {
   int dummy() { return 1; }
 }
```